service/gradle.lockfile

```diff
@@ -0,0 +1,250 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
@iliadsh 
Here is what is needed to set up dependency locking in the entire project 
example -> https://github.com/DataBiosphere/stairway

add file at root -> build.gradle with below 
```
tasks.named('dependencies') {
    dependsOn(':service:dependencies')
    dependsOn(':client:dependencies')
    dependsOn(':integration:dependencies')
}
```

add file at root
[gradle/dependency-locking.gradle](https://github.com/DataBiosphere/stairway/tree/develop/gradle)/dependency-locking.gradle with entire content

next in service/build.gradle, client/build.gradle and integration/build.gradle add single line at the end
`apply from: "$rootDir/gradle/dependency-locking.gradle"`

now you can simply run this from the root folder and it will lock dependencies in all sub projects
`./gradlew dependencies --write-locks`